dozer-ingestion/mysql/src/lib.rs

```diff
@@ -47,4 +47,7 @@ pub enum MySQLConnectorError {
 
     #[error("Failed to fetch query result. {0}")]
     QueryResultError(#[source] mysql_async::Error),
+
+    #[error("Schema had a breaking change: {0}")]
+    BreakingSchemaChange(String),
```

Comment:
I'd prefer if we use something like

```rust
enum BreakingSchemaChange {
    DatabaseDropped(String),
    ColumnRenamed {
        table_name: String,
        old_column_name: String,
        new_column_name: String,
    },
    ...
}
```

instead of a `String`.